src/collectors/netstat/test/testnetstat.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/python
+# coding=utf-8
+################################################################################
+
+from test import CollectorTestCase
+from test import get_collector_config
+from test import unittest
+from mock import patch
+
+from diamond.collector import Collector
+from netstat import NetstatCollector
+
+################################################################################
+
+
+class TestNetstatCollector(CollectorTestCase):
+    def setUp(self):
+        config = get_collector_config('NetstatCollector', {
+        })
+
+        self.collector = NetstatCollector(config, None)
+
+    @patch.object(Collector, 'publish')
+    def test(self, publish_mock):
+        NetstatCollector.PROC_TCP = self.getFixturePath('proc_net_tcp')
+        self.collector.collect()
+
+        metrics = {
+            'LISTEN':  9
```

Comment:
Can you expand this a bit? there is quite a few more metrics that should end up getting reported